test/jdk/java/util/zip/DeInflate.java

```diff
@@ -268,7 +273,7 @@ public static void main(String[] args) throws Throwable {
 
         byte[] dataIn = new byte[1024 * 512];
         rnd.nextBytes(dataIn);
-        byte[] dataOut1 = new byte[dataIn.length + 1024];
+        byte[] dataOut1 = new byte[dataIn.length + (32 * 1024)]; // See https://github.com/eclipse-openj9/openj9/issues/12740
```

Comment:
12.5% extra seems reasonable, but do we know what the maximum increase in size might be for the deflate algorithm?

Reply:
Note deflate is supposed to make it smaller.

Reply:
No compression algorithm can guarantee to make it smaller. The test appears to operate on random input so we should not be surprised that sometimes the 'compressed' data is larger, the question is how much larger?

Reply:
I tried various values, and the failing test showed how much data was re-inflated. With 16K it re-inflated 512675 (out of 524288), and with 8K extra it re-inflated 50494. The test runs twice with different data, and in both runs the numbers were similar though not exactly the same. Around + 28K should be good enough, but I allow for 32K.

Reply:
Lost a digit there, 8K was 504941 not 50494.